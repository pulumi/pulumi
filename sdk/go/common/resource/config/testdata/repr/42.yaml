my:outer:
  inner: -1
