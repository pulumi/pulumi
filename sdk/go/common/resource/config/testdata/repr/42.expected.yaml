decrypt:
  my:outer: '{"inner":-1}'
paths:
  my:outer:
    value:
      inner: -1
    string: '{"inner":-1}'
    redacted: '{"inner":-1}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAV/4ISAAEFaW5uZXIFaW50NjQEAgAB
    secure: false
    isObject: true
  my:outer.inner:
    value: "-1"
    string: "-1"
    redacted: "-1"
    object: DxAABnN0cmluZwwEAAItMQ==
    secure: false
    isObject: false
